stix_shifter_modules/crowdstrike_logscale/stix_translation/json/crowdstrikeedr_to_stix_map.json

```diff
@@ -0,0 +1,552 @@
+          "object": "behavior",
+          "references": "user"
+        },
+        {
+          "key": "process.creator_user_ref",
+          "object": "process",
+          "references": "user"
+        }
+      ],
+      "user_name": {
+        "key": "user-account.display_name",
+        "object": "user"
+      },
+      "GroupBehaviorsReferences": {
+        "key": "x-ibm-finding.x_behavior_refs",
+        "object": "finding",
+        "references": [
+          "behavior"
+        ],
+        "group_ref": true
+      },
+      "GroupTtpReferences": {
+        "key": "x-ibm-finding.ttp_tagging_refs",
+        "object": "finding",
+        "references": [
+          "ttp"
+        ],
+        "group_ref": "true"
+      }
+    },
+    "finding_type": {
+      "key": "x-ibm-finding.finding_type",
+      "object": "finding"
+    },
+    "behaviors_processed": {
+      "key": "x-ibm-finding.x_behaviors_processed",
+      "object": "finding"
+    },
+    "cid": [
+      {
+        "key": "x-oca-asset.x_cid",
+        "object": "asset"
+      }
+    ],
+    "created_timestamp": [
+      {
+        "key": "x-ibm-finding.time_observed",
+        "object": "finding",
+        "transformer": "LogscaleToTimestamp"
+      },
+      {
+        "key": "first_observed",
+        "transformer": "LogscaleToTimestamp"
+      }
+    ],
+    "detection_id": {
+      "key": "x-ibm-finding.name",
+      "object": "finding"
+    },
+    "date_updated": {
+      "key": "x-ibm-finding.x_last_updated",
+      "object": "finding"
+    },
+    "device": {
+      "agent_load_flags": {
+        "key": "x-crowdstrike-edr-agent.load_flags",
+        "object": "edr"
+      },
+      "agent_local_time": [
+        {
+          "key": "x-crowdstrike-edr-agent.local_time",
+          "object": "edr"
+        },
+        {
+          "key": "x-oca-asset.x_agent_ref",
+          "object": "asset",
+          "references": "edr"
+        }
+      ],
+      "agent_version": [
+        {
+          "key": "x-crowdstrike-edr-agent.version",
+          "object": "edr"
+        },
+        {
+          "key": "x-oca-asset.x_agent_ref",
+          "object": "asset",
+          "references": "edr"
+        }
+      ],
+      "bios_manufacturer": {
+        "key": "x-oca-asset.x_bios_manufacturer",
+        "object": "asset"
+      },
+      "bios_version": {
+        "key": "x-oca-asset.x_bios_version",
+        "object": "asset"
+      },
+      "config_id_base": {
+        "key": "x-crowdstrike-edr-agent.config_id_base",
+        "object": "edr"
+      },
+      "config_id_build": {
+        "key": "x-crowdstrike-edr-agent.config_id_build",
+        "object": "edr"
+      },
+      "config_id_platform": {
+        "key": "x-crowdstrike-edr-agent.config_id_platform",
+        "object": "edr"
+      },
+      "device_id": {
+        "key": "x-oca-asset.device_id",
+        "object": "asset"
+      },
+      "external_ip": [
+        {
+          "key": "ipv4-addr.value",
```

Comment:
Based on  our observation, the current mappings of IPv4 and IPv6 are getting mapped properly to either IPv4 or IPv6 correspondingly based on the values and even without transformer( since stix shifter automatically validates the values of IPv4 or IPv6 in accordance with the mappings).

Reply:
Can you add a unit test that confirms this? Maybe under 

"test_ipv4_addr_json_to_stix" you could add an example that checks that ipv6 is empty (or different from ipv4 if there is an ipv6).